bobweb/bob/openai_api_utils.py

```diff
@@ -130,7 +130,8 @@ def determine_suitable_model_for_version_based_on_message_history(version: str,
     """
     match version:
         case '3' | '3.5':
-            model = gpt_3_16k
+            # upgrade to 4 anyway
+            model = gpt_4o_mini
```

Comment:
Tässä tähän lohkoon ei enää mennä kun tota mallia ei voi valita komennossa. Sit taas jos voi, niin toi casen ehto pitäis muuttaa, että olis joku 'gptmini' tms se komento millä käyttäis tota.